prover/src/main.rs

```diff
@@ -99,6 +96,9 @@ struct Args {
 	/// The hex-encoded, bincode-serialized image id of the onchain program to prove
 	#[arg(short, long)]
 	image_id: String,
+	/// The Secret key of prover to sign and submit proof to chain.
+	#[arg(short, long, env)]
```

Comment:
Bit of a nit: taking it in through a cli arg could be insecure and might be an annoying ux if someone is running the app repeatedly. Maybe limit it to just the env var like:
```suggestion
	#[arg(env)]
```

Then I think we can just update the readme with the example